spec/views/accounts/_identity_verification.html.erb_spec.rb

```diff
@@ -118,6 +166,46 @@
           href: idv_in_person_ready_to_verify_url,
         )
       end
+
+      context 'with vtr values' do
+        let(:acr_values) { nil }
+        let(:vtr) { ['C2.P1'] }
+
+        it 'shows pending badge' do
+          expect(rendered).to have_content(t('account.index.verification.pending_badge'))
+        end
+
+        it 'shows content explaining that the user needs to finish verifying their identity' do
```

Comment:
Does 'references initiating SP in prompt instructing user to complete identity verification' read better?